### PR TITLE
chat test with VirtualTerminal

### PR DESCRIPTION
**Pull request by epoberezkin**

### CODE REVIEW

File: src/Simplex/Chat/Terminal.hs [hunk 1]
```diff
@@ -1,20 +1,26 @@
...
 {-# LANGUAGE ScopedTypeVariables #-}
 
 module Simplex.Chat.Terminal where
 
+import Control.Monad.Catch (MonadMask)
+import Control.Monad.IO.Class (MonadIO)
 import Simplex.Chat.Styled
 import Simplex.Chat.Types
 import System.Console.ANSI.Types
 import System.Terminal
```
Comment (efim-poberezkin) [inline src/Simplex/Chat/Terminal.hs:14]: import qualified? had to jump back and forth between github and ide to figure out what comes from where

File: tests/ChatTests.hs [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+aliceProfile :: Profile
+aliceProfile = Profile {contactRef = "alice", displayName = "Alice"}
+
+bobProfile :: Profile
+bobProfile = Profile {contactRef = "bob", displayName = "Bob"}
+
+testAddContact :: Spec
+testAddContact = describe "add chat contact" $
+  xit "add contact and send/receive message" $
```
Comment (efim-poberezkin) [inline tests/ChatTests.hs:24]: `it`

**Review by efim-poberezkin — APPROVED** [review]

File: tests/ChatTests.hs (hunk 2 above)
Comment (epoberezkin) [reply tests/ChatTests.hs:24]: it's disabled on purpose, it requires that the server is running and I can't easily start it in CI - it's in a different codebase, so some extra dance in CI is needed to run it here - not planning to do it right now.

File: src/Simplex/Chat/Terminal.hs (hunk 1 above)
Comment (epoberezkin) [reply src/Simplex/Chat/Terminal.hs:14]: maybe - will have a look - it's not this PR specifically that added it...

**Pull request merged**